.github/workflows/release-cli.yml

```diff
@@ -0,0 +1,98 @@
+name: Release new CLI versions
+on:
+  push:
+    branches: 
+      - "master"
+    paths:
+      - "cli/Makefile"
+env:
+  CLI_BUCKET: gs://cft-cli
+
+jobs:
+  release-new-cli:
+    name: Build and push new CFT dev tools image
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: "286.0.0"
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+
+      - name: Retrieve last released version
+        run: |-
+          LAST_VERSION=$(curl --silent "https://api.github.com/repos/GoogleCloudPlatform/cloud-foundation-toolkit/releases/latest" | \
+            grep '"tag_name":' |
+            sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
Can we not use `jq` instead which maybe cleaner if all we need is the value of `tag_name` like we do for TF latest release here https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/1f47e570882acc69ee9f7ce6a9c6a12149d6f82e/.github/workflows/update-tooling.yml#L17